indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -150,6 +185,11 @@ export default function FormItem({
       </>
     ) : null;
 
+  if (!show && !setupMode) {
+    form.change(rest.htmlName, null);
```

Comment:
Making such a call during a render is considered an error in react. Maybe you can move it into a useEffect if there's no better way to do this.

Reply:
Yep, indeed. I just changed it.